src/components/icon/test/resolveIcon.spec.ts

```diff
@@ -111,3 +115,20 @@ test('preload custom icon', async () => {
   const data = await resolveIcon('star');
   expect(data).toBe('<svg>Test</svg>');
 });
+
+test('load icons', async () => {
+  fetch.mockClear();
+
+  const cacheMap = getIconCacheMap();
+  cacheMap.clear();
+
+  expect(cacheMap.size).toBe(0);
```

Comment:
Do we need this? It seems we are just testing basic functionality of the datatyp "map"?